dim/src/routes/mod.rs

```diff
@@ -41,14 +42,19 @@ pub mod global_filters {
     use warp::reject;
     use warp::Rejection;
 
+    use std::borrow::Borrow;
     use std::convert::Infallible;
     use std::error::Error;
     use warp::Filter;
     use warp::Reply;
 
-    pub fn with_db(
-        conn: DbConnection,
-    ) -> impl Filter<Extract = (DbConnection,), Error = Infallible> + Clone {
+    pub fn with_db<C>(
+        conn: C,
+    ) -> impl Filter<Extract = (DbConnection,), Error = Infallible> + Clone + 'static
+    where
+        C: Borrow<DbConnection>,
```

Comment:
Why the borrow?

Reply:
I wanted to see if i didn't have to write `conn.clone()` everywhere